x-pack/plugins/actions/server/usage/task.ts

```diff
@@ -94,7 +94,7 @@ export function telemetryTaskRunner(logger: Logger, core: CoreSetup, kibanaIndex
 
 function getNextMidnight() {
   return moment()
-    .add(1, 'd')
-    .startOf('d')
+    .add(1, 's')
+    .startOf('s')
```

Comment:
Was this change intentional?

Reply:
ops, will revert this changes. Thanks for catch!